### PR TITLE
show referral ID

### LINKED ISSUE #726: Show referral ID before stream start
Right now if you're referred by someone the Refer page will show you as "Organic" even when the referralId cookie is set. This issue is to change this so that if the cookie is set (i.e. referralId=carl) then show who referred you even before opening a stream.

Currently, people who coming in via referral links look at this refer page to see if that referral worked. So even if the referral id cookie is set, it shows organic and they get confused. Once you open a stream via a referral, then this will show that you were referred by someone.
Screen Shot 2022-06-13 at 10 42 55 AM

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
Fixes #726

### CODE REVIEW

File: src/containers/main/ReferContainer/index.tsx [hunk 1]
```diff
@@ -31,15 +31,26 @@ export const ReferContainer: React.FC<IProps> = () => {
...
 				const affiliate = await contractMethods.customerToAffiliate(address.toLowerCase()).call();
 				if (affiliate !== '0') {
-					const referral = await contractMethods.affiliates(affiliate).call();
-					if (isMounted) {
-						setReferredBy(referral.name);
-					}
+					// const referral = await contractMethods.affiliates(affiliate).call();
+					// if (isMounted) {
+					// 	setReferredBy(referral.name);
+					// }
```
Comment (mikeghen) [inline src/containers/main/ReferContainer/index.tsx]: if this chunk of code isn't needed, remove it

**Review by mikeghen — CHANGES_REQUESTED** [review]

File: src/containers/main/ReferContainer/index.tsx (hunk 1 above)
Comment (mikeghen) [reply src/containers/main/ReferContainer/index.tsx]: I don't think you can get rid of this like this, I visited the page with an account that was already referred by another affiliate and it shows me that I'm organic even though Im not, its because you just commented this out. If someone was already referred, youre code will just show them as organic unless the cookie is set, if the cookie is set, it will show the referral id of the wrong affiliate. 

TLDR, you don't account for the senario someone was already referred, commenting this out isn't the right thing to do to resolve this issue

**Review by mikeghen — CHANGES_REQUESTED** [review]

**Review by mikeghen — APPROVED** [review]
